frontend/src/app/selectors/experiment.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
```

Comment:
Would be nice to add a little unit test for this new refactored module, since we have the beginnings of a working suite